app/resources/views/pages/profile/profile.blade.php

```diff
@@ -40,6 +39,11 @@
 		</form>
  	</div>
     <div class="col-md-9 col-sm-8 col-xs-12">
+		@if (Session::has('message'))
+			<div class="alert alert-success alert-dismissible" role="alert" style="margin-top: 16px">
```

Comment:
Move inline styles like your "margin-top" to the appropriate SCSS file.  I think resources/assets/sass/_profile.scss is right for this.